.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
     -   id: check-merge-conflict
     -   id: check-added-large-files
 -   repo: https://github.com/psf/black
-    rev: stable
+    rev: 23.9.1
```

Comment:
I was getting all sort of issues installing pre-commit, suggestion from the tool was to use auto-update. So here we are